packages/forms/src/Components/Repeater.php

```diff
@@ -196,6 +199,18 @@ public function getAddAction(): Action
         return $action;
     }
 
+    public function addActionAlignment(Alignment | string | Closure | null $addActionAlignment): static
+    {
+        $this->addActionAlignment = $addActionAlignment;
+
+        return $this;
+    }
+
+    public function getAddActionAlignment(): Alignment | string | null
+    {
+        return $this->evaluate($this->addActionAlignment);
```

Comment:
Please add some handling here to check if the string is a valid `Alignment` option and transform it if it is, else it can remain as a string